static/js/snap-pages/components/NewSnapNotification/NewSnapNotification.tsx

```diff
@@ -0,0 +1,40 @@
+import { Link, Notification } from "@canonical/react-components";
+import { ISnap } from "../../types";
+
+export const NewSnapNotification = ({ snap }: { snap: ISnap }) => {
```

Comment:
I think this should be `function NewSnapNotifiction` and the export at the bottom of the file to match other React components